include/ray.h

```diff
@@ -11,6 +11,8 @@ class Ray {
   bool is_inside_object = false; // if inside an object
 
 public:
+  float importance;
```

Comment:
Does it make sense to have importance public? Either way we now have 2 importance variables in this file :) Need to remove one